core/examples/luigi-sample-vue/README.md

```diff
@@ -4,46 +4,24 @@
 
 This is the Vue.JS-based sample application which runs with Luigi framework.
 
->**NOTE:** The authorization flow in this application is a mock implementation for local development. **DO NOT USE IN PRODUCTION!**
 
 ## Development
 
-Linking and building with [Lerna](https://lerna.js.org) is the preferred development method. This monorepo uses Lerna for project management. 
 
 To run this application, follow these steps:
 
-1. Ensure that you have installed Lerna. If not, run the following command.
-    
+1. Install dependencies.
     ```bash
-    npm install -g lerna
-    ```
-
-2. Install dependencies.
-    ```bash
-    # The `lerna bootstrap` command executes the Node Package Manager (NPM) installation and links cross-dependencies.
-
-    lerna bootstrap
+    npm install
```

Comment:
Maybe we can mention in what directory the commands have to be run just to avoid confusion? (Ignore this if it's too obvious 😅)

Reply:
:-) They should be executed in the directory where this README.md is. I would say that is clear.